packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -192,12 +192,15 @@ class ArtifactStore {
     }
   }
 
-  static void ensureHasSkyEnginePackage() {
-    Directory skyEnginePackage = new Directory(path.join(packageRoot, 'sky_engine'));
-    if (!skyEnginePackage.existsSync()) {
+  static void validateSkyEnginePackage() {
+    if (engineRevision == null) {
       printError("Cannot locate the sky_engine package; did you include 'flutter' in your pubspec.yaml file?");
       throw new ProcessExit(2);
     }
+    if (engineRevision != expectedEngineRevision) {
+      printError("Error: incompatible sky_engine package; please run 'pub get flutter' to get the correct one.\n");
```

Comment:
Instead of `'pub get flutter'` I think we want to say `'pub get'` (or `'flutter upgrade'`?).